src/k8s-extension/HISTORY.rst

```diff
@@ -7,6 +7,7 @@ Release History
 ++++++++++++++++++
 * microsoft.azureml.kubernetes: Disable service bus by default, do not create relay for managed clusters.
 * microsoft.azureml.kubernetes: Rename inferenceLoadBalancerHA to inferenceRouterHA and unify related logic.
+* microsoft.azureml.kubernetes: Added private arguments for directly passing ssl key/cert base64 string
```

Comment:
Please add this as a release note for version `1.2.3`. `1.2.2` is already released